tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.h

```diff
@@ -141,6 +141,7 @@ extern ST_DBG_IP_DESIGN_INFO g_std_dbg_ip_info;
 
 #define ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK 0x20
 #define ST_DBG_IP_CONFIG_H2T_T2H_RESET_FIELD 0x1
+#define ST_DBG_IP_CONFIG_H2T_T2H_RESET_FIELD 0x1
```

Comment:
Isn't this the same definition as on the previous line?  We should remove it for clarity.

Reply:
fixed , removed duplicate  #define